src/GafferArnoldUI/LightBlockerVisualiser.cpp

```diff
@@ -173,7 +173,7 @@ void setFalloffGroupSettings( IECoreGL::Group *group, const IECore::CompoundData
 // LightBlockerVisualiser implementation.
 //////////////////////////////////////////////////////////////////////////
 
-class GAFFERSCENEUI_API LightBlockerVisualiser : public LightFilterVisualiser
+class LightBlockerVisualiser : public LightFilterVisualiser
```

Comment:
@johnhaddon Think there is where the `Export.h` import originally snuck in from. Have fixed this properly now.